flutter-studio/src/io/flutter/FlutterStudioInitializer.java

```diff
@@ -25,7 +25,7 @@ public void run() {
     ApplicationInfo info = ApplicationInfo.getInstance();
     if ("Google".equals(info.getCompanyName())) {
       String version = info.getFullVersion();
-      if (version.startsWith("2.") || (version.contains("Beta") && !version.endsWith("7"))) {
+      if (version.startsWith("2.")) {
```

Comment:
This likely address https://github.com/flutter/flutter-intellij/issues/1824 ?

Reply:
Yes. I updated the intro to make that clear.